app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt

```diff
@@ -265,5 +274,5 @@ class OnlineLibraryFragment : BaseFragment(), FragmentActivityExtensions {
     .apply {
       onSelectAction = ::storeDeviceInPreferences
     }
-    .show(requireFragmentManager(), getString(R.string.pref_storage))
+    .show(parentFragmentManager, getString(R.string.pref_storage))
```

Comment:
It's good you wanna remove the deprecated method/s. but it's off-topic here. Create an issue for this and then change theire.